packages/config/config/devices/0x0431/ecodim.json

```diff
@@ -101,29 +110,29 @@
 			"description": "Enable or Disable external switch to pair network. Default value: 0.",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
```

Comment:
Description says "default 0", line 114 says "default 1".

Reply:
Changed it